test/test-style.js

```diff
@@ -2,6 +2,17 @@
 const fs = require('fs');
 const path = require('path');
 
+function escapeInvisibles(str) {
+  const invisibles = [['\b', '\\b'], ['\f', '\\f'], ['\n', '\\n'], ['\r', '\\r'], ['\v', '\\v'], ['\0', '\\0']];
```

Comment:
This might be more readable:
```suggestion
  const invisibles = [
    ['\b', '\\b'],
    ['\f', '\\f'],
    ['\n', '\\n'],
    ['\r', '\\r'],
    ['\v', '\\v'],
    ['\0', '\\0'],
  ];
```

Reply:
Good idea! Thank you!